modules/preview-react/side-panel/lib/SidePanel.tsx

```diff
@@ -206,14 +207,27 @@ const SidePanel = ({
   );
 };
 
-export type ToggleButtonProps = Omit<IconButtonProps, 'aria-label'>;
+export type ToggleButtonProps = Omit<IconButtonProps, 'aria-label'> & {
+  /**
+   * The tooltip label for when the side panel is collapsed
+   * @default 'Expand'
+   */
+  expandLabel?: string;
```

Comment:
Not really a label. What do you think about `tooltipTextExpand` or `tooltipTextCollapsed`. I'm not sure `expand` or `collapsed` should be used here. The description mentions the current state (collapsed) while the prop name references the text (expand)